packages/@aws-cdk/aws-ecs-patterns/lib/base/application-load-balanced-service-base.ts

```diff
@@ -108,6 +108,13 @@ export interface ApplicationLoadBalancedServiceBaseProps {
    */
   readonly loadBalancer?: ApplicationLoadBalancer;
 
+  /**
+   * Listener port of the application load balancer that will serve traffic to the service.
+   *
+   * @default - defaults to 80 if not specified
```

Comment:
Please provide the default as only the value. 

`@default 80`

Reply:
fixed